docs/reference/content/driver-async/getting-started/quick-tour-admin.md

```diff
@@ -1,236 +0,0 @@
-administrative features available in the driver.
-
-The following code snippets come from the `QuickTourAdmin.java` example code
-that can be found with the [driver
-source]({{< srcref "driver-async/src/examples/tour/QuickTourAdmin.java">}}).
-
-{{% note %}}
-See the [installation guide]({{< relref "driver-async/getting-started/installation-guide.md" >}})
-for instructions on how to install the MongoDB Driver.
-{{% /note %}}
-
-## Setup
-
-To get started we'll quickly connect and create a `mongoClient`, `database` and `collection`
-variable for use in the examples below:
-
-```java
-MongoClient mongoClient = new MongoClient(new ConnectionString("mongodb://localhost"));
-MongoDatabase database = mongoClient.getDatabase("mydb");
-MongoCollection<Document> collection = database.getCollection("test");
-```
-
-{{% note %}}
-Calling the `getDatabase()` on `MongoClient` does not create a database.
-Only when a database is written to will a database be created.  Examples include the creation of an index or the insertion of a document 
-into a previously non-existent collection.
-{{% /note %}}
-
-{{% note %}}
-Sometimes you will need the same or similar callbacks more than once.  In these situations
-it makes sense to DRY (Do not Repeat Yourself) up your code and save the callback either
-as a concrete class or assign to a variable as below:
-
-```java
-SingleResultCallback<Void> callbackWhenFinished = new SingleResultCallback<Void>() {
-    @Override
-    public void onResult(final Void result, final Throwable t) {
-        System.out.println("Operation Finished!");
-    }
-};
-```
-{{% /note %}}
-
-
-## Get A List of Databases
-
-You can get a list of the available databases:
-
-```java
-mongoClient.listDatabaseNames().forEach(new Block<String>() {
-    @Override
-    public void apply(final String s) {
-        System.out.println(s);
-    }
-}, callbackWhenFinished);
-```
-
-## Drop A Database
-
-You can drop a database by name using a `MongoClient` instance:
-
-```java
-mongoClient.getDatabase("databaseToBeDropped").drop(callbackWhenFinished);
-```
-
-## Create A Collection
-
-Collections in MongoDB are created automatically simply by inserted a document into it. Using the 
-[`createCollection`]({{< apiref "com/mongodb/async/client/MongoDatabase.html#createCollection-java.lang.String-com.mongodb.async.SingleResultCallback-">}}) 
-method, you can also create a collection explicitly in order to customize its configuration. For example, to create a capped collection sized to 1 megabyte:
-
-```java
-database.createCollection("cappedCollection",
-  new CreateCollectionOptions().capped(true).sizeInBytes(0x100000),
-  callbackWhenFinished);
-```
-
-## Get A List of Collections
-
-You can get a list of the available collections in a database:
-
-```java
-database.listCollectionNames().forEach(new Block<String>() {
-    @Override
-    public void apply(final String databaseName) {
```

Comment:
Change from databaseName to name or collectionName (name is used in the sync playbook)